src/status_im/chat/handlers.cljs

```diff
@@ -28,16 +28,18 @@
     (messages/delete-by-chat-id id)))
 
 (defn delete-messages!
-  [{:keys [current-chat-id]} [_ chat-id]]
-  (let [id (or chat-id current-chat-id)]
-    (messages/delete-by-chat-id id)))
+  [{:keys [current-chat-id chats]} [_ chat-id]]
+  (let [id                   (or chat-id current-chat-id)
+        {:keys [group-chat]} (chats/get-by-id chat-id)]
+    (when group-chat
+      (messages/delete-by-chat-id id))))
 
 (defn delete-chat!
   [_ [_ chat-id]]
-  (let [{:keys [debug? group-chat]} (chats/get-by-id chat-id)]
-    (if (and (not debug?) group-chat)
-      (chats/set-inactive chat-id)
-      (chats/delete chat-id))))
+  (let [{:keys [debug?]} (chats/get-by-id chat-id)]
+    (if debug?
+      (chats/delete chat-id)
```

Comment:
Not sure delete and set-inactive should be different

Reply:
I'm also curious about this and `debug?` flag purpose. Not sure i can answer why it's there. My intention on this change was to set 1-1 chats as inactive, not delete them.